src/python/nimbusml/internal/core/preprocessing/normalization/lpnormalizer.py

```diff
@@ -0,0 +1,93 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------------------------
+# - Generated by tools/entrypoint_compiler.py: do not edit by hand
+"""
+LpNormalizer
+"""
+
+__all__ = ["LpNormalizer"]
+
+
+from ....entrypoints.transforms_lpnormalizer import transforms_lpnormalizer
+from ....utils.utils import trace
+from ...base_pipeline_item import BasePipelineItem, DefaultSignature
+
+
+class LpNormalizer(BasePipelineItem, DefaultSignature):
+    """
+    **Description**
+        Normalize vectors (rows) individually by rescaling them to unit norm (L2, L1 or LInf). Performs the following operation on a vector X: Y = (X - M) / D, where M is mean and D is either L2 norm, L1 norm or LInf norm.
+
```

Comment:
Strange, should this line be automatically broken down to smaller ones once run_autopep is run?

Reply:
I checked autopep8 runs but doesnt fix this. so its by design

---
In reply to: [319709441](https://github.com/microsoft/NimbusML/pull/253#discussion_r319709441) [](ancestors = 319709441)